Week3/Homework/Project/Credit-Card-Validator.js

```diff
@@ -0,0 +1,90 @@
+function cardChecker(cardNumber) {
+
+    if (cardNumber * 10 % 2 !== 0) {        // Check if cardNumber is not a number (if it has letters or symbols)
+
+        const newCardNumber = prompt("Invalid Characters, Please Enter Just Numbers :");
+        return cardChecker(newCardNumber);
+
+    }
+
+    else if (cardNumber * 10 % 2 === 0) {       // Check if cardnumber is a number
```

Comment:
You don't really need `else-if` here I think as a simple `else` block would also work fine. Since the condition `cardNumber * 10 % 2` can only result in two values: `=== 0` or `!== 0`, you can use a simple `if-else` block.